config/campaignSets.json

```diff
@@ -47,18 +47,5 @@
 			{ "x":34,  "y":417, "file":"DATA/FINAL.H3C",    "image":"CAMPUA1", "video":"UNHOLY",  "open": true },
 			{ "x":404, "y":414, "file":"DATA/SECRET.H3C",  "image":"CAMPSP1", "video":"SPECTRE", "open": true }
 		]
-	},
-	"wog" : 
-	{
-		/// wog campaigns, currently has no assigned button in campaign screen and thus unused
-		"images" : [ {"x": 0, "y": 0, "name":"CAMPZALL"} ],
-		"exitbutton" : {"x": 658, "y": 482, "name":"CMPSCAN", "hotkey" : 27},
-		"items":
-		[
-			{ "x":90,  "y":72,  "file":"DATA/ZC1.H3C", "image":"CAMPZ01", "open": true},
-			{ "x":539, "y":72,  "file":"DATA/ZC2.H3C", "image":"CAMPZ02", "open": true},
-			{ "x":43,  "y":245, "file":"DATA/ZC3.H3C", "image":"CAMPZ03", "open": true},
-			{ "x":311, "y":242, "file":"DATA/ZC4.H3C", "image":"CAMPZ04", "open": true}
-		]
```

Comment:
Please, keep, as it may be useful. Having this in config will help a lot if someone decide to implement

Reply:
Not against keeping it, but might make more sense to have this in wog mod. Highly doubt that wog campaigns would ever be supported or if there will be any interest in them.

Can keep it but in commented-out form.

Reply:
certainly in WOG mod, but as for me removing this config should go together with adding it to WOG. Otherwise risk to lose it forever is high (as we lost WOG translations - old translations removed while new not added)

Reply:
Sorry, but how can you lose something forever with version control systems?

Reply:
Restoring in commented-out form, will be moved to wog later with #1276 

Still don't see any need for this though - not something that will ever be used & we have git history to find old stuff.